pyecharts/charts/basic_charts/polar.py

```diff
@@ -79,4 +80,34 @@ def add(
                     "itemStyle": itemstyle_opts,
                 }
             )
+        if label_opts.get("show"):
+            self.options.get("series").append(
+                {
+                    "type": ChartType.CUSTOM,
+                    "coordinateSystem": "polar",
+                    # TODO: 提供自定义 function
+                    "renderItem": JsCode(
+                        """function(params, api) {
+                        var values = [api.value(0), api.value(1)];
+                        var coord = api.coord(values);
+                        return {
+                            type: 'text',
+                            position: [3 * Math.sin(coord[3]), 3 * Math.cos(coord[3])],
+                            rotation: coord[3] + Math.PI / 2,
+                            origin: [coord[0], coord[1]],
+                            style: {

```

Comment:
hard coded style will become an issue. someone else may want different style.

Reply:
Yeap, so i left a `TODO` tag there

Reply:
Let’s build quality in.